src/Symfony/Component/Mime/Security/SMimeEncrypter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Security;
+
+use Symfony\Component\Mime\Exception\RuntimeException;
+use Symfony\Component\Mime\Message;
+use Symfony\Component\Mime\WrappedMessage;
+
+/**
+ * @author Sebastiaan Stok <s.stok@rollerscapes.net>
+ *
+ * @experimental in 4.3
+ */
+final class SMimeEncrypter extends SMime
+{
+    private $certs;
+    private $cipher;
+
+    /**
+     * @param string|string[] $certificate Either a lone X.509 certificate, or an array of X.509 certificates
+     */
+    public function __construct($certificate, int $cipher = OPENSSL_CIPHER_AES_256_CBC)
+    {
+        if (\is_array($certificate)) {
+            $this->certs = array_map([$this, 'normalizeFilePath'], $certificate);
+        } else {
+            $this->certs = $this->normalizeFilePath($certificate);
+        }
+
+        $this->cipher = $cipher;
+    }
+
+    public function encryptMessage(Message $message): WrappedMessage
```

Comment:
Could be `encrypt()` as the name of the class is already clear enough I think?

Reply:
Yes.